samples/package.json

```diff
@@ -7,11 +7,11 @@
         "test": "jest"
     },
     "dependencies": {
-        "axios": "^0.21.1",
+        "axios": "^0.21.4",
```

Comment:
@tnorling Is the package-lock needed for this, or can we remove it like we do for other samples?